images/fcos-base-image/README.md

```diff
@@ -30,3 +29,7 @@ When possible, prefer not to change the `/etc` files, but the ones in `/usr/`. T
 managed by rpm-ostree through a three-way merge, and any files modified by the user will not be updated by rpm-ostree
 when the OS is updated.
 
+The directories `/etc` and `/var` are mounted as read-write which lets users write and modify files.
+
+The directory `/etc` may be changed by deployments, but will not override user made changes. 
+The content under `/var` is left untouched by rpm-ostree when applying upgrades or rollbacks.
```

Comment:
Also add that we should not consider adding content under /var in the Container image (as it gets ignored).